draft-ietf-ppm-dap.md

```diff
@@ -1264,30 +1265,37 @@ enum {
 } PrepareStepState;
 
 struct {
-  ReportID report_id;
   PrepareStepState prepare_step_state;
   select (PrepareStep.prepare_step_state) {
-    case continued: opaque prep_msg<0..2^32-1>; /* VDAF preparation message */
+    case continued: opaque prep_msg<0..2^32-1>; /* VDAF preparation message or share */
     case finished:  Empty;
     case failed:    ReportShareError;
   };
 } PrepareStep;
 
 struct {
-  PrepareStep prepare_steps<1..2^32-1>;
-} AggregationJobResp;
+  ReportID report_id;
+  PrepareStep init_prep_step;
+  PrepareStep stepped_prep_step;
+} ReportAggregationInitResp;
+
+struct {
+  ReportAggregationInitResp report_shares<1..2^32-1>;
+} AggregationJobInitResp;
 ~~~
 
-The message is a sequence of PrepareStep values, the order of which matches that
-of the ReportShare values in `AggregationJobInitReq.report_shares`. Each report
-that was marked as invalid is assigned the `PrepareStepState` `failed`.
-Otherwise, the `PrepareStep` is either marked as continued with the output
-`prep_msg`, or is marked as finished if the VDAF preparation process is finished
-for the report share. The Helper's aggregation job is now in round 0.
+The message is a sequence of `ReportAggregationInitResp` values, the order of
+which matches that of the ReportShare values in
+`AggregationJobInitReq.report_shares`. Each report that was marked as invalid is
+assigned the `PrepareStepState` `failed`; if the prepare step corresponding to
+`init_prep_step` fails, `stepped_prep_step` is unspecified. Otherwise, the
+`PrepareStep`s are either marked as continued with the output `prep_msg`, or are
+marked as finished if the VDAF preparation process is finished for the report
+share. The Helper's aggregation job is now in round 0.
 
 On success, the helper responds to the leader with HTTP status code 201 Created
-and a body consisting of the `AggregationJobResp`, with media type
-"application/dap-aggregation-job-resp".
+and a body consisting of the `AggregationJobInitResp`, with media type
+"application/dap-aggregation-job-init-resp".
```

Comment:
What's the reason for changing AggregationJobResp to AggregationJobInitResp?

Reply:
(this no longer applies with the new "ping-pong" approach)